fortuna/src/marketplace/ChassisView.js

```diff
@@ -0,0 +1,13 @@
+//@flow strict
+import * as React from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
So with the way that our web page is functioning, as long as this CSS import is included in Login.js or App.js, it will be imported on all of our js files. This renders the import of the bootstrap CSS file here unnecessary.